data/json/items/comestibles.json

```diff
@@ -3087,6 +3087,7 @@
         "addiction_type" : "mutagen",
         "spoils_in" : 0,
         "use_action" : "PURIFIER",
+		"flags" : "PURIFIER",
```

Comment:
Tabs.
Before committing, try to find all tabs in your edited files. All good text editors have search+replace and can detect tabs.